app/javascript/components/subject/LastDateExposure.js

```diff
@@ -104,6 +104,7 @@ class LastDateExposure extends React.Component {
 
   createModal(title, message, toggle, submit) {
     message += this.props.has_group_members ? '(s):' : '.';
+    const update_continuous_exposure = title === 'Continuous Exposure';
```

Comment:
That seems a little brittle - thoughts on elevating label to a param instead?

Reply:
Yeah I agree it isn't my favorite. The issue with adding label as a param is that there is more than one label, and I have a feeling that could grow in the future. I could instead add a boolean param for if this is updating LDE or continuous exposure that would at least be better than string comparison.